doc/environ.sh.sample

```diff
@@ -177,6 +177,16 @@ export KOS_CFLAGS="${KOS_CFLAGS} -fno-builtin"
 #
 export KOS_SH4_PRECISION="-m4-single-only"
 
+# Use LRA (Local Register Allocator) pass
+#
+# Uncomment this line to use the modern Local Register Allocator pass during
+# code generation instead of the default older reload pass. This option is
+# known to be unstable or less performant for SH at this time, but will likely
+# become mandatory in future versions of GCC.
```

Comment:
```suggestion
# become mandatory in future versions of GCC, so feel free to help us test.
```
Long shot in hell, but maybe some good-guy DC scener might see this and want to help log shit to BugZilla for the upcoming register allocator D-day...